README.md

```diff
@@ -150,20 +150,50 @@ We collaborate closely with the design and product team. The design team provide
 
 ## Deployment
 
+### Git Branching Model
+
+- When a Github issue is started, create a branch with the `feature/` prefix. Every push to branches with this prefix will automatically deploy to the review AWS environment. See the [Review branches](#review-branches) section for details on how to access this build.
+
+- After a pull request for a `feature/` branch is approved, it is merged into the `staging` branch. This will trigger an automatic deployment to the staging AWS environment. Note that changes may take several minutes to take effect. The staging app is accessible at https://staging.fightpandemics.work.
+
+- When we are ready to release to production, a pull request is opened to merge the `staging` branch to `production`. After this pull request is approved and merged, it will trigger an automatic deployment to the production AWS environment. Note that changes may take several minutes to take effect. The production app is accessible at https://fightpandemics.com
+
+- A `hotfix/` branch is created if there is a critical fix that needs to be deployed to production as soon as possible, but staging has untested code that is not safe to deploy to production.
+    - This branch must be created off the `production` branch.
+    - Every push to `hotfix/` branches will automatically deploy to the review AWS environment. See the [Review branches](#review-branches) section for details on how to access this build.
+    - Two pull requests must be opened for hotfixes: one PR to merge into `production` and one PR to backmerge `production` into `staging` after the initial PR is merged to `production`.
+    - Hotfixes must be tested in the staging AWS environment before being merged to the `production` and `staging` branches. In order to deploy only the hotfix to the staging enviroment, create and push a git tag with a `hotfix-` prefix. This will trigger a deployment to the staging environment: `git tag -a hotfix-<GITHUB_ISSUE_NUMBER> && git push origin hotfix-<GITHUB_ISSUE_NUMBER>`.
```

Comment:
```suggestion
    - Hotfixes must be tested in the review AWS environment before being merged to the `production` and `staging` branches. In order to deploy only the hotfix to the staging enviroment, create and push a git tag with a `hotfix-` prefix. This will trigger a deployment to the staging environment: `git tag -a hotfix-<GITHUB_ISSUE_NUMBER> && git push origin hotfix-<GITHUB_ISSUE_NUMBER>`.
```

Besides this substitution I think the docs and workflow files look good. Thanks @mannykary !

Reply:
@josh It's actually staging. That's the reason why I added the tag trigger for `hotfix-` so that we can deploy any arbitrary branch the staging environment. We wouldn't want to test the hotfix in the review environment as it isn't as "fully baked" as staging.

Reply:
Okay. In that case, the order of action as well as rationale for this is not quite clear. It reads like it's a) deployed to review environment (for testing I would assume?), b) PR is merged into `production` c) PR is backmerged into `staging`. But somewhere in between it is also pushed to `staging` environment. After what and why is not quite clear... How is not quite "fully baked"?

Reply:
Let me know if this is clearer: https://github.com/FightPandemics/FightPandemics/pull/1269/commits/be41a69d7c0fe0dc36eea2181e1bc128c086eb6e

When I say the review environment is not "fully baked", I mean that I don't consider the review environment to be a valid environment for QA testing for the following reasons:
- All the review builds read from and write to the same database, and also use the same Auth0 tenant, so data integrity is not really guaranteed
- It does not quite have the same infrastructure as the staging and production environments - the way it's set up is the load balancer forwards requests to the appropriate Fargate task by looking at the subdomain of the URL. This does not exist in staging/production.
- There is a cleanup job that kills review builds that haven't been deployed to in more than a day, so that could potentially throw off QA testing. I anticipate adding more cleanup jobs to rein in our resource usage so as to minimize expenses
- This hasn't been implemented yet, but I plan to vertically scale down the Fargate tasks to provision a lot less memory

The review builds are really more of a convenience for developer testing rather than official QA testing. That is not to say that we can't do ANY QA testing in the review build. I just wouldn't consider review build QA testing as the final green light to merge into the `production` branch and deploy to production. The staging environment mirrors the production environment more closely, so it would be more prudent to test in staging rather than the review environment.

Reply:
Thanks for clarifying @mannykary